x-pack/legacy/plugins/code/public/components/integrations/code_flyout.scss

```diff
@@ -1,29 +1,21 @@
-.code-flyout {
-
-}
-
-.code-flyout .code-flyout-header {
-
-}
-
-.code-flyout .code-flyout-header .code-flyout-header-file {
+.codeFlyout .codeFlyout__header .codeFlyout__header--file {
   position: absolute;
   display: flex;
   align-items: center;
   z-index: 1;
 }
 
-.code-flyout .code-flyout-header .code-flyout-header-file .euiIcon {
+.codeFlyout .codeFlyout__header .codeFlyout__header--file .euiIcon {
```

Comment:
@daveyholler is the hierarchy here really necessary? With BEM you really shouldn't have to scope things further than a single class

Reply:
also curious if you saw https://github.com/elastic/kibana/pull/46884/files#r334066227, and what your thoughts were there.

Reply:
Yeah, the hierarchy probably is not necessary. I just did a find/replace, but I'm sure that could be cleaned up more.